packages/get-platform/src/getPlatform.ts

```diff
@@ -74,15 +74,6 @@ export async function getos(): Promise<GetOSResult> {
   const distroInfo = await resolveDistro()
   const archFromUname = await getArchFromUname()
 
-  // TODO: add 'arm64' to the `[...].includes(arch)` check once we have arm64 engines for Alpine
```

Comment:
I removed this altogether instead of doing what TODO says because we don't have this check for other platforms, and I don't think it's generally useful to hard error on unsupported CPU architecture because users can totally build from source for 32-bit ARM or RISC-V or whatever they need.

Reply:
Note: we no longer "hard error", we warn now

Reply:
Nice, I missed that, thanks for pointing this out!

Reply:
Should we still have the check for the CPU architecture and just remove the distro from it then?

Reply:
Let's go with `if (platform === 'linux' && !['x64', 'arm64'].includes(arch)) {`

Drop the `alpine:` prefix from the `alpine:unsupported-arch` warning and change the first sentence to `Prisma only officially supports Linux on `amd64` (x86_64) and `arm64` (aarch64) system architectures`, please.

Reply:
`platform === 'linux'` check was not necessary here since the whole code path here is only relevant for Linux, other than that, done